testonly/hammer/maphammer/main.go

```diff
@@ -45,6 +45,7 @@ var (
 	metricsEndpoint = flag.String("metrics_endpoint", "", "Endpoint for serving metrics; if left empty, metrics will not be exposed")
 	seed            = flag.Int64("seed", -1, "Seed for random number generation")
 	operationsFlag  = flag.Uint64("operations", ^uint64(0), "Number of operations to perform")
+	checkSigsFlag   = flag.Bool("check_signatures", true, "If false SignedMapHead signatures will be ignored")
```

Comment:
(nit, copy-of-pre-existing) Prefer not to have `...Flag` in the name of flags.

Reply:
Just being consistent with the preexisting style :)

Reply:
Yeah, fine to leave as-is -- I'll try to get around to an overall sweep at some point (cf. [c-t-go pull #86](https://github.com/google/certificate-transparency-go/pull/86))

Reply:
I've done it now :)

Reply:
Thanks for fixing the others.